packages/slice-machine/lib/builders/SliceBuilder/index.tsx

```diff
@@ -87,30 +61,23 @@ const SliceBuilder: React.FC<SliceBuilderProps> = ({
     }
   }, [data]);
```

Comment:
hmm, adding an object to evaluate :/

Reply:
at some point I guess? But it's not for today!